roles/matrix-jitsi/tasks/util/setup_jitsi_auth.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+#
+# Tasks related to setting up Jitsi authentication mechanisms
+#
+
+- name: Ensure matrix-jitsi-prosody container is running
+  systemd:
+    state: started
+    name: matrix-jitsi-prosody
+
+- name: Ensire Jitsi internal authentication users are created
+  shell: "docker exec matrix-jitsi-prosody prosodyctl --config /config/prosody.cfg.lua register {{ item.username | quote }} meet.jitsi {{ item.password | quote }}"
```

Comment:
What happens if you call this again and again for the same username? Does it fail with an error the 2nd time around?

What happens if you call this again for the same username, but with a different password? Does the password get changed? If not, we should document it next to the `matrix_jitsi_prosody_auth_internal_accounts` variable definition.

Reply:
I'll run those tests and update the documentation if necessary. Good catch!

Reply:
After testing, it seems that the default behavior of "prosodyctl register" is to modify a user if it exists. I also made sure there is no duplicate user with the old password.